modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsRepository.java

```diff
@@ -365,6 +265,23 @@ public void saveLocalStatistics(StatisticsKey key, ObjectStatisticsImpl statisti
         locStats.put(key, statistics);
     }
 
+    /**
+     * Clear specified partition ids statistics.
+     *
+     * @param key Key to remove statistics by.
+     * @param partsToRemove Set of parititon ids to remove.
+     */
+    public void clearLocalPartitionIdsStatistics(StatisticsKey key, Set<Integer> partsToRemove) {
```

Comment:
```suggestion
    public void clearLocalStatistics(StatisticsKey key, Set<Integer> partsToRemove) {
```

Reply:
Remove Ids, keep Partition. Local statistics also are stored by the StatisticsRepository but these method clears partition statistics.